config/environments/development.rb

```diff
@@ -65,8 +65,6 @@
     s3_region: 'us-east-2'
   }
 
-  Aws.config.update({
-    region: 'us-east-1',
-    credentials: Aws::Credentials.new(AppConfig["aws"]["access_token"], AppConfig["aws"]["secret_token"])
-  })
+  Aws.config.update(region: 'us-east-1',
```

Comment:
AWS is an acronym, can we add it to inflector config and make it `AWS`?

Reply:
That's how [aws/aws-sdk-ruby](https://github.com/aws/aws-sdk-ruby) named everything. Kinda inclined to leave it that way, even if it looks weird.

Reply:
:+1: :(